resources/microvm-kernel-config

```diff
@@ -1455,7 +1488,19 @@ CONFIG_VIRTIO_CONSOLE=y
 # CONFIG_SPI is not set
 # CONFIG_SPMI is not set
 # CONFIG_HSI is not set
-# CONFIG_PPS is not set
+CONFIG_PPS=y
```

Comment:
I see that before it was not set. I guess regenerating the kernel config has set it on. I am wondering, besides drivers which are used by Firecracker, is there another criteria which come into play when adding drivers in the Firecracker example kernel config for amd64? Is the `CONFIG_PPS=y` worth to add?

Reply:
According to make menuconfig this option was automatically selected.
 

> Symbol: PPS [=y]                                                                                                                                                                                    
>    Type  : tristate                                                                                                                                                                                    
>    Prompt: PPS support           
>        .....                                                                                                                                                                       
>        Selected by: PTP_1588_CLOCK [=y] && NET [=y] && POSIX_TIMERS [=y]

Reply:
> According to make menuconfig this option was automatically selected.
> 
> > Symbol: PPS [=y]
> > Type  : tristate
> > Prompt: PPS support
> > .....
> > Selected by: PTP_1588_CLOCK [=y] && NET [=y] && POSIX_TIMERS [=y]

I think we can let it through. The kernel defaults should do if it is not something flagrant.